src/Controller/CRUDController.php

```diff
@@ -61,6 +61,29 @@ public function setContainer(ContainerInterface $container = null)
         $this->configure();
     }
 
+    public function __call($method, $arguments)
+    {
+        if ('render' === $method) {
+            if (count($arguments) < 1) {
+                throw new \LogicException(sprintf(
+                    '%s::%s requires at least one argument',
+                    __CLASS__,
+                    $method
+                ));
+            }
+            @trigger_error(sprintf(
```

Comment:
You should avoid `sprintf` for performance, use string concatenation instead.

Reply:
I would prefer readability over performance here